src/app/content/practiceQuestions/selectors.ts

```diff
@@ -53,3 +53,8 @@ export const selectedSectionHasPracticeQuestions = createSelector(
   practiceQuestionsSummary,
   (section, summary) => Boolean(section && summary && pageHasPracticeQuestions(section.id, summary))
 );
+
+export const currentSectionPracticeQuestions = createSelector(
```

Comment:
maybe name it `questionsForCurrentSection`? I was confused after seeing `currentSection` at the beginning

Reply:
we actually don't ever need to return the whole set of questions, at some point we'll have the index in the state and we'll just return the one question that is currently selected, but i think that is part of another card so i don't know if we have to do any state stuff here, we can just pass some stubbed data into the `Question` component